<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userMan</class>
 <widget class="QWidget" name="userMan">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>714</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
                font-size:24px;
                font-family:sans-serif;
                }
                Form{
                background-color:white;
                }
                QFrame{
                background:rgba(0,0,0,0.8);
                border-radius:15px;
                }
                QPushButton{
                background:#03a9f4;
                color:#fff;
                border-radius:15px;
                }
                QLineEdit{
                border-radius:15px;
                color:#03a9f4;
                }
                QLabel{
                color:#fff;
                background:transparent;
                font-size:30px;
                }
            </string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>260</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="adduserButton">
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:2px groove gray;
</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deluserButton">
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:2px groove gray;
</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeauthButton">
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:2px groove gray;
</string>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>用户信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>81</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userID"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>密码：  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userPswd"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>权限：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="userAuth">
        <property name="styleSheet">
         <string notr="true">border-radius:5px;
border:2px groove gray;
</string>
        </property>
        <item>
         <property name="text">
          <string>admin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>user</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
